app/posts/find-teacher-training/2021-08-18-new-international-students-section.md

```diff
@@ -0,0 +1,21 @@
+---
+title: New ‘International students’ section
+description: Why we have added a new section to course paes
+date: 2021-08-18
+screenshots:
+  items:
+    - text: International students section – provider can sponsor visas
+      src: can-sponsor-visas.png
+    - text: International students section – provider unable to sponsor visas
+      src: unable-to-sponsor-visas.png
+---
+
+We have started looking at how to improve the experience of non-UK based candidates applying for postgraduate teacher training. As part of this, we reviewed the information on course pages.
```

Comment:
Suggestions:
1. Replace 'We have' with 'We've'
2. Replace 'of' with 'for' in 'improve the experience of non-UK based candidates...'